src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -1770,11 +1797,11 @@ public static XmlReader Create(string inputUri)
         // Creates an XmlReader according to the settings for parsing XML from the given Uri.
         public static XmlReader Create(string inputUri, XmlReaderSettings settings)
         {
-            return XmlReader.Create(inputUri, settings, (XmlParserContext)null);
+            return XmlReader.Create(inputUri, settings, (XmlParserContext?)null);
         }
 
         // Creates an XmlReader according to the settings and parser context for parsing XML from the given Uri.
-        public static XmlReader Create(string inputUri, XmlReaderSettings settings, XmlParserContext inputContext)
+        public static XmlReader Create(string inputUri, XmlReaderSettings settings, XmlParserContext? inputContext)
         {
             settings ??= XmlReaderSettings.s_defaultReaderSettings;
             return settings.CreateReader(inputUri, inputContext);
```

Comment:
For below all overloads `XmlReaderSettings settings, XmlParserContext inputContext` are nullable, probably baseUri too